Fix failing coverageReport

<!-- Checklist for reporting a bug
- Check the issue tracker. The issue you have may have already been reported.
- Ensure you have tested the latest developers version.
-->

**Describe the bug**
<!-- A clear and concise description of what the bug is. -->
The coverage report is failing for all builds.

**Expected behavior**
<!-- A clear and concise description of what you expected to happen. -->
It should pass.

**Steps to reproduce the behavior:**
1. Open a new PR & the build will pass but the coverage report will fail

**Environment**
- Version of Kiwix Android : master

**Logs**
<!-- If applicable, add logs to help the developers in identifying your problem otherwise delete this heading. -->
```
Please provide the repository token to upload reports via `-t :repository-token`
##[error]Codecov failed with the following error: The process 'bash' failed with exit code 1
```

--- a/.github/workflows/coverage.yml
+++ b/.github/workflows/coverage.yml
@@ -35,7 +35,5 @@ jobs:
         run: ./gradlew jacocoTestDebugUnitTestReport jacocoTestCustomExampleDebugUnitTestReport
 
       - name: Upload coverage to Codecov
-        uses: codecov/codecov-action@v1
-        with:
-          token: ${{ secrets.CODECOV_TOKEN }}
-          fail_ci_if_error: true
+        run: |
+          bash <(curl -s https://codecov.io/bash) -t ${{ secrets.CODECOV_TOKEN }}

